notebooks/stage/watson_keywords.ipynb

```diff
@@ -0,0 +1,1355 @@
+{
```

Comment:
<p>Can we load these values from environment if they are not defined or it's executed from automation?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/mailing-list-analysis-toolkit/pull/42/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>